lib/training_data.py

```diff
@@ -1,32 +1,36 @@
 import cv2
 import numpy
+from random import shuffle
 
 from .umeyama import umeyama
 
-random_transform_args = {
-    'rotation_range': 10,
-    'zoom_range': 0.05,
-    'shift_range': 0.05,
-    'random_flip': 0.4,
-}
-
-def get_training_data(images, batch_size):
-    indices = numpy.random.randint(len(images), size=batch_size)
-    for i, index in enumerate(indices):
-        image = images[index]
-        image = random_transform(image, **random_transform_args)
-        warped_img, target_img = random_warp(image)
-
-        if i == 0:
-            warped_images = numpy.empty(
-                (batch_size,) + warped_img.shape, warped_img.dtype)
-            target_images = numpy.empty(
-                (batch_size,) + target_img.shape, warped_img.dtype)
-
-        warped_images[i] = warped_img
-        target_images[i] = target_img
-
-    return warped_images, target_images
+def read_image(fn, random_transform_args=random_transform_args):

```

Comment:
the default random_transform_args is not accessible from this context and is not passed from the method that is calling it.

Reply:
yes, I moved the args in the model as the GAN has other params. But I messed up things when rebasing...